hazelcast-client/src/main/resources/hazelcast-client-config-3.5.xsd

```diff
@@ -51,7 +51,7 @@
         </xs:complexType>
     </xs:element>
     <xs:complexType name="aws">
-        <xs:sequence>
+        <xs:all>
```

Comment:
This kind of replacement seems relaxing xmls so enabling user to get rid of order restriction. That is ok if we don't need to have them in order.